packages/blade/src/components/Input/BaseInput/StyledBaseInput.web.tsx

```diff
@@ -104,12 +104,18 @@ export const StyledBaseInput = React.forwardRef<
       numberOfLines,
       type,
       hasPopup,
+      shouldIgnoreBlur,
+      setShouldIgnoreBlur,
       ...props
     },
     ref,
   ) => {
     const commonProps = {
       onBlur: (event: React.ChangeEvent<HTMLInputElement>): void => {
+        if (shouldIgnoreBlur && setShouldIgnoreBlur) {
```

Comment:
when do we set this as true?

Reply:
In actionlistitem when someone clicks on item.

This is the same flag that we handle to ignore the dropdown's blur when click in somewhere inside the dropdown itself